code/parachain/runtime/picasso/src/lib.rs

```diff
@@ -246,30 +246,32 @@ impl assets_registry::Config for Runtime {
 	type ForeignAssetId = composable_traits::xcm::assets::XcmAssetLocation;
 	type UpdateAssetRegistryOrigin = EnsureRootOrTwoThirdNativeCouncil;
 	type ParachainOrGovernanceOrigin = EnsureRootOrTwoThirdNativeCouncil;
-	type CurrencyFactory = CurrencyFactory;
 	type WeightInfo = weights::assets_registry::WeightInfo<Runtime>;
+	type Convert = ConvertInto;
 }
 
 parameter_types! {
 	pub PabloPalletId: PalletId = PalletId(*b"pal_pblo");
 	pub TWAPInterval: u64 = (MILLISECS_PER_BLOCK as u64) * 10;
+	pub LPTokenExistentialDeposit: Balance = 10_000;
 }
 
 impl pablo::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type AssetId = CurrencyId;
 	type Balance = Balance;
 	type Convert = sp_runtime::traits::ConvertInto;
-	type CurrencyFactory = CurrencyFactory;
 	type Assets = Assets;
+	// TODO(Connor): Won't impact current pools, must implement new assts system
```

Comment:
```suggestion
	// TODO(Connor): Won't impact current pools since this is only required for pool creation, must implement new assts system before any more pools are created
```